uui/settings.ts

```diff
@@ -1,315 +1,1148 @@
+    dropdownIcon: Icon;
+}
+
+interface LinkButtonSizes {
+    default: LinkButtonProps['size'];
+    defaultWeight: LinkButtonProps['weight'];
+}
+
+interface LinkButtonSettings {
+    icons: LinkButtonIcons;
+    sizes: LinkButtonSizes;
+}
+
+const linkButtonSettings: LinkButtonSettings = {
+    icons: {
+        dropdownIcon: DropdownIcon,
+    },
+    sizes: {
+        default: '36',
+        defaultWeight: 'semibold',
+    },
+};
+
+interface ModalIcons {
+    closeIcon: Icon;
+}
+
+interface ModalSizes {
+    defaultWidth: number;
+    defaultMaxHeight: CSSProperties['height'];
+    defaultMobileMaxHeight: CSSProperties['height'];
+    defaultHeaderPadding: FlexRowProps['padding'];
+    defaultFooterColumnGap: FlexRowProps['columnGap'];
+    defaultFooterPadding: FlexRowProps['padding'];
+    defaultFooterVPadding: FlexRowProps['vPadding'];
+}
+
+interface ModalSettings {
+    icons: ModalIcons;
+    sizes: ModalSizes;
+}
+
+const modalSettings: ModalSettings = {
+    icons: {
+        closeIcon: CrossIcon,
+    },
+    sizes: {
+        defaultWidth: 420,
+        defaultMaxHeight: '80dvh',
+        defaultMobileMaxHeight: '100dvh',
+        defaultHeaderPadding: '24',
+        defaultFooterColumnGap: '12',
+        defaultFooterPadding: '24',
+        defaultFooterVPadding: '24',
+    },
+};
+
+interface NotificationCardIcons {
+    closeIcon: Icon;
+    hintIcon: Icon;
+    successIcon: Icon;
+    warningIcon: Icon;
+    errorIcon: Icon;
+}
+
+interface NotificationCardSizes {
+    action: LinkButtonProps['size'];
+}
+
+interface NotificationCardSettings {
+    icons: NotificationCardIcons;
+    sizes: NotificationCardSizes;
+}
+
+const notificationCardSettings: NotificationCardSettings = {
+    icons: {
+        closeIcon: CrossIcon,
+        hintIcon: HintIcon,
+        successIcon: SuccessIcon,
+        warningIcon: WarningIcon,
+        errorIcon: ErrorIcon,
+    },
+    sizes: {
+        action: '30',
+    },
+};
+
+interface NumericInputIcons {
+    arrowIcon: Icon;
+}
+
+interface NumericInputSizes {
+    default: NumericInputProps['size'];
+}
+
+interface NumericInputSettings {
+    icons: NumericInputIcons;
+    sizes: NumericInputSizes;
+}
+
+const numericInputSettings: NumericInputSettings = {
+    icons: {
+        arrowIcon: DropdownIcon,
+    },
+    sizes: {
+        default: '36',
+    },
+};
+
+interface PaginatorIcons {
+    leftArrowIcon: Icon;
+    rightArrowIcon: Icon;
+}
+
+interface PaginatorSizes {
+    default: ButtonProps['size'];
+}
+
+interface PaginatorSettings {
+    icons: PaginatorIcons;
+    sizes: PaginatorSizes;
+}
+
+const paginatorSettings: PaginatorSettings = {
+    icons: {
+        leftArrowIcon: LeftArrowIcon,
+        rightArrowIcon: RightArrowIcon,
+    },
+    sizes: {
+        default: '36',
+    },
+};
+
+interface PickerInputIcons {
+    toggler: {
+        clearIcon: Icon;
+        dropdownIcon: Icon;
+    },
+    body: {
+        selectIcon: Icon;
+        boldSelectIcon: Icon;
```

Comment:
let's remove boldSelectionIcon in flavor to callback: 
selectIcon: Icon | (size: PickerInputProps['size']) => Icon

in component:
        const SelectIcon = typeof settings.pickerInput.icons.body.selectIcon === "function" ? settings.pickerInput.icons.body.selectIcon(props.size) : settings.pickerInput.icons.body.selectIcon;